include/oneapi/dpl/pstl/glue_algorithm_ranges_impl.h

```diff
@@ -41,8 +332,8 @@ any_of(_ExecutionPolicy&& __exec, _Range&& __rng, _Predicate __pred)
 {
     const auto __dispatch_tag = oneapi::dpl::__ranges::__select_backend(__exec, __rng);
 
-    return oneapi::dpl::__internal::__ranges::__pattern_any_of(__dispatch_tag, ::std::forward<_ExecutionPolicy>(__exec),
-                                                               views::all_read(::std::forward<_Range>(__rng)), __pred);
+    return oneapi::dpl::__internal::__ranges::__pattern_any_of(__dispatch_tag, std::forward<_ExecutionPolicy>(__exec),
```

Comment:
The same comment like in another file. I would not remove `::` without any other meaningful change. It just creates a lot of noise.

Reply:
reverted

Reply:
Now it's been reverted.